scripts/dev-setup.sh

```diff
@@ -25,18 +29,32 @@ if ! sudo -n true; then
     exit 1
 fi
 
-# Make sure Docker Debian package isn't installed
-if dpkg -s docker.io &>/dev/null; then
-    echo "Docker is already installed from Debian's repository. Please uninstall it first."
-    exit 1
-fi
+# Make sure OS-provided Docker package isn't installed
+case "$DISTRO" in
+    Debian-*)
+        if dpkg -s docker.io &>/dev/null; then
+            echo "Docker is already installed from Debian's repository. Please uninstall it first."
+            exit 1
+        fi
+    ;;
+    RedHatEnterprise-8.*) true ;; # Not in OS repositories.
+    *) echo "Internal error: $DISTRO not matched in case block." && exit 1 ;;
+esac
 
 # Upgrade everything
 # (this ensures that an existing Docker CE installation is up to date before continuing)
-sudo apt-get update && sudo apt-get upgrade -y
+case "$DISTRO" in
+    Debian-*)             sudo apt-get update && sudo apt-get upgrade -y ;;
+    RedHatEnterprise-8.*) sudo dnf upgrade -y ;;
+    *) echo "Internal error: $DISTRO not matched in case block." && exit 1 ;;
+esac
 
 # Install dependencies
-sudo apt-get -y install bash-completion git git-review vim
+case "$DISTRO" in
+    Debian-*)             sudo apt-get -y install bash-completion git git-review vim ;;
```

Comment:
We may want to consider adding kubectl to this list as well.  It's mentioned in DEVELOP.md but doesn't get installed by anything in this script.

Reply:
The install script adds an alias: https://github.com/certusone/wormhole/blob/885754e723cc7b9c136ceb37471ece8b50e9cff6/scripts/dev-setup.sh#L93

Reply:
Ah, I see.  I use fish so bash aliases have no effect for me.  I'll look more carefully next time.